metricflow/specs.py

```diff
@@ -303,13 +313,14 @@ def qualified_name(self) -> str:  # noqa: D
         return self.element_name
 
 
-class OrderBySpec(FrozenBaseModel):  # noqa: D
+@dataclass(frozen=True)
+class OrderBySpec:  # noqa: D
```

Comment:
We don't serialize this? I guess not....

Reply:
Not right now, but we will soon, so added it.